sample/Pizzeria.Store.Web/States/Orders/Actions/AddPizzaToOrderAction.cs

```diff
@@ -1,3 +1,5 @@
+using System.Diagnostics.CodeAnalysis;
 namespace Pizzeria.Store.Web.States.Orders.Actions;
 
+[SuppressMessage("Performance", "CA1515:Consider making public types internal", Justification = "Fluxor actions must be public")]
```

Comment:
This suppression shouldn't be needed as there's section in .editorconfig for *.razor.cs and this rule is turned off